Resources/Storage/XROOTStorage.py

```diff
@@ -1597,3 +1598,32 @@ def getCurrentURL( self, fileName ):
     cwdUrl = result['Value']
     fullUrl = '%s/%s' % ( cwdUrl, fileName )
     return S_OK( fullUrl )
+
+  def constructURLFromLFN( self, lfn, withWSUrl = False ):
```

Comment:
I am worried that someone will change the original constructURLFromLFN and break things in Xroot